security/integration/main.go

```diff
@@ -225,70 +293,64 @@ func runCAForNodeAgentTests() error {
 	}
 
 	err = waitForNodeAgentCertificateUpdate(fmt.Sprintf("http://%v:%v",
-		na_service.Status.LoadBalancer.Ingress[0].IP, 8080))
+		naService.Status.LoadBalancer.Ingress[0].IP, 8080))
 	if err != nil {
 		return fmt.Errorf("failed to check certificate update node-agent (err: %v)", err)
-	} else {
-		glog.Info("Certificate of NodeAgent was updated and verified successfully")
-	}
-
-	// Delete created services
-	err = utils.DeleteService(opts.clientset, opts.namespace, ca_service.GetName())
```

Comment:
Why do you remove this?

Reply:
Moved into cleanUpCAForNodeAgentTests function

Reply:
I see. Thanks.